pom.xml

```diff
@@ -23,6 +23,10 @@
         <junit.version>5.4.2</junit.version>
         <aws.sdk.version>1.11.591</aws.sdk.version>
         <skipITs>true</skipITs>
+        <kotlin.version>1.3.31</kotlin.version>
```

Comment:
any reason not to use 1.3.41? (and spotless 1.23.1, ktlint 0.33.0)

Reply:
no reason... does dependabot not handle that?

Reply:
indeed it might, though dependabot doesn't catch everything (and I happened to notice those three)